### DISCUSSION TITLE
目前的GPT形式下能导入role:system 吗

### DISCUSSION THREAD

**Discussion by A2Sumie** [Tsuk1ko/CQ-picfinder-robot#412] [一般讨论]
https://platform.openai.com/docs/guides/chat/introduction
目前附加参数的写法里可以预先导入这个系统信息吗（？）
抄个作业

**Comment by Tsuk1ko** [comment]
可以，就写就行了，就是官方文档里面那种写法

**Comment by Tsuk1ko** [comment]
哦，不是写到附加参数，是 prependMessages
